core/views.py

```diff
@@ -55,3 +59,45 @@ def search(request, parent_page_id=None):
             'page_type_string': page_type_string,
         })
     )
+
+
+def site_search(request, template=None, results_per_page=10, path=None):
+    # Get default templates
+    if template is None:
+        if hasattr(settings, 'WAGTAILSEARCH_RESULTS_TEMPLATE'):
+            template = settings.WAGTAILSEARCH_RESULTS_TEMPLATE
+        else:
+            template = 'wagtailsearch/search_results.html'
+
```

Comment:
Wouldn't it be more pythonic to simply access the setting and catch the AttributeError instead?